_source/_posts/2022-05-15-jhipster-ionic-spring-boot.adoc

```diff
@@ -0,0 +1,481 @@
+---
+layout: blog_post
+title: "Integrate Ionic and Spring Boot Securely with JWT and OAuth 2.0"
+author: matt-raible
+by: advocate
+communities: [mobile,java,javascript]
+description: "Build an Ionic frontend for your Spring Boot backend, all using the power of JHipster!"
+tags: [ionic, spring-boot, spring-security, jhipster]
+tweets:
+- "Create an Ionic app with a Spring Boot backend in minutes with the new Ionic blueprint for @jhipster!"
+- "Introducing JHipster Ionic: 'Wow! Did you really just build that?!'"
+- "Ionic is awesome. Learn how you to create apps quickly using this powerful framework. 💪"
+image:
+type: conversion
+github: https://github.com/oktadev/okta-jhipster-ionic-example
+---
+:page-liquid:
+:toc: macro
+:experimental:
+:commandkey: &#8984;
+
+[comment]
+--
+todo: research and figure out best title.
+Other options:
+
+- Integrate Ionic and Spring Boot with OAuth/OIDC
+- Integrate Ionic and Spring Boot
+- Ionic for JHipster
+--
+
+Ionic allows you to create mobile apps with web technologies that look and act like native apps.
+Because they're built with web technologies (HTML, JavaScript, and CSS), you can also deploy your Ionic apps as webapps.
```

Comment:
```suggestion
Because they're built with web technologies (HTML, JavaScript, and CSS), you can also deploy your Ionic apps as web apps.
```
web app still looks wrong to me as one word, but YMMV...